tests/e2e/specs/Modeler.spec.js

```diff
@@ -0,0 +1,216 @@
+// const mountVue = require('cypress-vue-unit-test');
+// import ModelerApp from '../../../src/ModelerApp';
+
+const generateXML = (nodeName) => {
+  return `<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_03dabax" targetNamespace="http://bpmn.io/schema/bpmn" exporter="ProcessMaker Modeler" exporterVersion="1.0">
+  <bpmn:process id="Process_1" isExecutable="true">
+    <bpmn:startEvent id="node_2" name="${nodeName}" />
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1">
+      <bpmndi:BPMNShape id="node_2_di" bpmnElement="node_2">
+        <dc:Bounds x="150" y="150" width="36" height="36" />
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</bpmn:definitions>`;
+};
+
+const dragFromSourceToDest = (source, dest, position) => {
+  const dataTransfer = new DataTransfer();
+  cy.get(source).trigger('dragstart', { dataTransfer });
+  cy.get(dest).trigger('dragenter');
+  cy.get(dest).trigger('drop', { offsetX: position.x, offsetY: position.y });
+};
+
+function coordinateGenerator() {
```

Comment:
Could you add a comment for this function? Or, better yet, it may benefit from a slight refactor. It seems it's always used the same way:

```js
{ x: coordinateGenerator(), y: coordinateGenerator() }
```

It could be renamed to `generateRandomPoint` and return an object with the shape `{ x: Number, y: Number }`. And possibly add constraints based on the paper dimensions, as `500` and `150` are magic numbers.